Resources/public/js/apps/ez-platformuiapp.js

```diff
@@ -733,6 +739,13 @@ YUI.add('ez-platformuiapp', function (Y) {
                     view: 'studioPlusPresentationView',
                     callbacks: ['open', 'checkUser', 'handleSideViews', 'handleMainView']
                 }, {
+                    name: "translateContent",
```

Comment:
I have added translateContent route config in shell.html.twig: https://github.com/ezsystems/PlatformUIBundle/pull/280/files#diff-def2dab3cc9208048b5a2e96663033a7R75